tests/h/emails/reply_notification_test.py

```diff
@@ -155,15 +155,6 @@ def test_returns_parent_email_as_recipients(self, notification, pyramid_request)
 
         assert recipients == ["pat@ric.ia"]
 
-    def test_calls_token_serializer_with_correct_arguments(
-        self, notification, pyramid_request, token_serializer
-    ):
-        generate(pyramid_request, notification)
-
-        token_serializer.dumps.assert_called_once_with(
-            {"type": "reply", "uri": "acct:patricia@example.com"}
-        )
```

Comment:
There is no token serializer to call any more. These are good examples of the atomized tests which in their worst incarnation cause a test per line of code under test.

These are nearly impossible to follow to understand the flow of the code, and nearly impossible to spot logical gaps in.